tiatoolbox/models/architecture/hovernetplus.py

```diff
@@ -38,16 +38,13 @@ class HoVerNetPlus(HoVerNet):
 
     HoVer-Net+ takes an RGB input image, and provides the option to simultaneously
     segment and classify the nuclei present, aswell as semantically segment different
-    regions or layers in the images.
+    regions or layers in the images. Note HoVer-Net+ trained models are currently at
+    20X magnification, in contrast to HoVer-Net at 40X magnification.
```

Comment:
Be consistent about using power or mpp. The yaml file mentions mpp, so this should be mpp. Also, rather than the model being trained at this magnification, its the entire design of the arch and its post processing / preprocing etc.

Reply:
Edited to address this.